api/api-pastis/pastis-commons/src/main/java/fr/gouv/vitamui/pastis/common/dto/jaxb/BaliseXML.java

```diff
@@ -149,6 +150,16 @@ private static void setValueAndDataRNG(ElementProperties node, boolean presenceC
             dataRNG.setDataType(RNGConstants.getTypesMap().get(node.getName()).getLabel());
         }
 
+        if (
+            (node.getName() != null &&
+                ((node.getName().equals("CodeListVersions") && presenceChildrenNode) || presenceChildrenNode) &&
```

Comment:
Peut être simplifié par :
```suggestion
                presenceChildrenNode &&
```
Idem quelques lignes au-dessus